### PR TITLE
Add push-artifact command

### LINKED ISSUE #9: push-artifact
Add a command to download artifacts in current taskgroup and push them on the repo with skopeo.

From #8

### PR DESCRIPTION
**Pull request by La0**
Fixes #9 

Without any argument, while running on Taskcluster, it will push all public images from dependant tasks

### CODE REVIEW

**Comment by La0** [comment]
Successful push on https://tools.taskcluster.net/groups/IO7wltVmSZycacQV2ZxKEQ/tasks/bNA8ZBL-RVaUJm33bQW7ew/details

File: taskboot/docker.py [hunk 1]
```diff
@@ -1,16 +1,37 @@
...
+    '''
+    def __init__(self, binary):
+        # Check the tool is available on the system
+        self.binary = shutil.which(binary)
+        assert self.binary is not None, 'Binary {} not found in PATH'.format(binary)
+
+    def run(self, command, **params):
+        command = [self.binary] + command
+        out = subprocess.run(command, **params)
+        assert out.returncode == 0, 'Tool failure for: {}'.format(' '.join(command))
```
Comment (marco-c) [inline taskboot/docker.py]: You could do subprocess.run(..., check=True) to make the assertion automatic

Comment (marco-c) [reply taskboot/docker.py]: (Nit)

File: taskboot/utils.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import requests
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+def download_progress(url, path):
```
Comment (marco-c) [inline taskboot/utils.py]: Given that this is downloading from S3, it might be worth retrying in case of an intermittent failure

Comment (marco-c) [reply taskboot/utils.py]: (if you don't do it in this PR, please file an issue so we don't forget)

Comment (La0) [reply taskboot/utils.py]: :+1:

**Pull request merged**